app/src/main/java/com/sdp/movemeet/Navigation/Navigation.java

```diff
@@ -43,4 +43,5 @@ public static void goToChat(View view) {
         Intent intent = new Intent(mContext, ChatActivity.class);
         mContext.startActivity(intent);
     }
+
```

Comment:
Just for information, I had to modify this Navigation class to set the "chat ID" to the "general_chat" as follows:

public static void goToChat(View view) {
        Context mContext = view.getContext();
        Intent intent = new Intent(mContext, ChatActivity.class);
        intent.putExtra("ACTIVITY_CHAT_ID", "general_chat");
        mContext.startActivity(intent);
    }

The purpose was here to be able to access the general chat directly by clicking on the chat button of your side nav panel.

Reply:
Perfect thanks, I just modified it !